datasets/moroco/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Source Data
+
+
+#### Data Collection and Normalization
+
+For the data collection, five of the most popular news websites in Romania and the Republic of Moldova were targetted. Given that the data set was obtained through a web scraping technique, all the HTML tags needed to be removed, as well as replace consecutive white spaces with a single space. 
+
+As part of the pre-processing, we remove named entities, such as country names, cities, public figures, etc. The named entities have been replaced with $NE$. The necessity to remove them, comes also from the scope of this dataset: categorization by topic. Thus, the authors decided to remove named entities in order to prevent classifiers from taking the decision based on features that are not truly indicative of the topics. 
+
+#### Who are the source language producers?
+
+The original text comes from news websites from Romania and the Republic of Moldova.
+
+### Annotations
+
+#### Annotation process
+
+As mentioned above, MOROCO is composed of text samples from the top five most popular news websites in Romania and the Republic of Moldova, respectively. Since there are topic tags in the news websites targetd, the text samples can be automatically labeled with the corresponding category.
+
+#### Who are the annotators?
+
+N/A
+
+### Personal and Sensitive Information
+
+The textual data collected for MOROCO consists in news reports freely available on the Internet and of public interest. 
+To the best of authors' knowledge, there is no personal or sensitive information that needed to be considered in the said textual inputs collected.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+This dataset is part of an effort to encourage text classification research in languages other than English. Such work increases the accessibility of natural language technology to more regions and cultures. 
+In the past three years there was a growing interest for studying Romanian from a Computational Linguistics perspective. However, we are far from having enough datasets and resources in this particular language.
+
+### Discussion of Biases
+
+The data included in MOROCO spans a well defined time frame of a few years. Part of the topics that were of interest then in the news landscape, might not show up nowadays or a few years from now in news websites.
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+Published and managed by Radu Tudor Ionescu and Andrei Butnaru.
+
+### Licensing Information
+
+CC BY-SA 4.0 License
+
+### Citation Information
+
+@inproceedings{ Butnaru-ACL-2019,
+    author = {Andrei M. Butnaru and Radu Tudor Ionescu},
+    title = "{MOROCO: The Moldavian and Romanian Dialectal Corpus}",
+    booktitle = {Proceedings of ACL},
+    year = {2019},
+    pages={688--698},
+}
```

Comment:
````suggestion
```
@inproceedings{ Butnaru-ACL-2019,
    author = {Andrei M. Butnaru and Radu Tudor Ionescu},
    title = "{MOROCO: The Moldavian and Romanian Dialectal Corpus}",
    booktitle = {Proceedings of ACL},
    year = {2019},
    pages={688--698},
}
```
````
add code block